### PR TITLE
LPS-200590 Success message does not appear correctly when adding a field

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPS-200590](https://liferay.atlassian.net/browse/LPS-200590)

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -7245,6 +7245,7 @@ field-reference-serves-as-a-frienldy-identifier=Field reference serves as a frie
 field-set=Field Set
 field-sets=Field Sets
 field-settings=Field Settings
+field-successfully-added=Field successfully added.
```
Comment (carolmariaabb) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7248]: @diogomelo09 please change to `The field was successfully added.`

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (diogomelo09) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties]: Done!! :heavy_check_mark:

**Comment by diogomelo09** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**